cms-plugin/src/main/java/com/agiletec/plugins/jacms/aps/system/services/contentpagemapper/ContentPageMapperManager.java

```diff
@@ -69,6 +70,10 @@ public String getPageCode(String contentId) {
 
 	@Override
 	public void updateFromPageChanged(PageChangedEvent event) {
+		if (_logger.isDebugEnabled()) {
```

Comment:
can we change from _logger to logger and use Slf4j or @Slf4j ?

Reply:
Done